python-spec/src/somacore/spatial.py

```diff
@@ -528,23 +528,29 @@ def create(
         uri: str,
         *,
         type: pa.DataType,
-        reference_level_shape: Sequence[int],
-        axis_names: Sequence[str] = ("c", "y", "x"),
+        shape: Sequence[int],
+        coordinate_space: Union[Sequence[str], coordinates.CoordinateSpace] = (
+            "x",
+            "y",
+        ),
         axis_types: Sequence[str] = ("channel", "height", "width"),
```

Comment:
Is there a better name or better argument to take to distinguish between the on disk data order and the coordinate space?